api-operator/pkg/controller/api/api_controller.go

```diff
@@ -216,6 +216,22 @@ func (r *ReconcileAPI) Reconcile(request reconcile.Request) (reconcile.Result, e
 	mgwDockerImage.RepositoryName = dockerRegistryConf.Data[repositoryNameConst]
 	operatorMode := controlConfigData[operatorModeConst]
 
+	//optional get push docker registry configs
+	dockerPushRegistryConf := k8s.NewConfMap()
+	if controlConfigData[dockerPushRegName] != "" {
+		k8s.Get(&r.client, types.NamespacedName{Namespace: config.SystemNamespace, Name: controlConfigData[dockerPushRegName]},
```

Comment:
formatting issue

Reply:
This code does not exist in the updated `1.2.x` branch[1]
[1] https://github.com/wso2/k8s-api-operator/blob/1.2.x/api-operator/pkg/controller/api/api_controller.go#L222